### PR TITLE
Add a government schema

### PR DESCRIPTION
**Pull request by cbaines**
In preparation for Whitehall publishing governments to the Publishing API.

### CODE REVIEW

**Comment by cbaines** [comment]
This depends on https://github.com/alphagov/content-data-api/pull/1367

File: formats/government.jsonnet [hunk 1]
```diff
@@ -0,0 +1,25 @@
+(import "shared/default_format.jsonnet") + {
+  document_type: "government",
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        start_date: {
```
Comment (thomasleese) [inline formats/government.jsonnet]: Could we call this `started_on` and the other one `ended_on` for consistency with similar naming in role appointments?

Comment (cbaines) [reply formats/government.jsonnet]: Yeah, that sounds fine :+1:

Comment (cbaines) [reply formats/government.jsonnet]: I've updated this Pull Request, and the govuk-content-schemas change now.

**Pull request merged**